api/envoy/config/listener/v3/listener.proto

```diff
@@ -318,4 +318,8 @@ message Listener {
   // Enable MPTCP (multi-path TCP) on this listener. Clients will be allowed to establish
   // MPTCP connections. Non-MPTCP clients will fall back to regular TCP.
   bool enable_mptcp = 30;
+
+  // Whether the listener should limit connections based upon the value of
+  // `global_downstream_max_connections <https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/overload_manager/overload_manager#limiting-active-connections>`_
```

Comment:
I think an rst anchor next to the relevant sub-section in the doc (`docs/root/configuration/operations/overload_manager/overload_manager.rst`), and reference that here, just in case the comment will move somewhere else.
Same comment for the bootstrap.proto.